public/app/plugins/datasource/elasticsearch/types.ts

```diff
@@ -12,7 +12,7 @@ export type Interval = 'Hourly' | 'Daily' | 'Weekly' | 'Monthly' | 'Yearly';
 
 export interface ElasticsearchOptions extends DataSourceJsonData {
   timeField: string;
-  esVersion: number;
+  esVersion: string;
```

Comment:
it describes the "after" format. in the PR description I also mentioned this problem and I think it would be out of scope for this PR, but I definitely want to iterate over this and have different typings for "whatever might come in" and "what we actually work with internally"

Reply:
@Elfo404 just make sure you don't forget that this was once a number :)